src/components/singleBook.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+function SingleBook(props) {
+  const { book } = props;
+  return (
+    <li className="SingleBook">
+      <h3>
+        {' '}
+        {book.title}
+        {' '}
+      </h3>
```

Comment:
¿Es necesario el espacio entre el `h3` y el título? ¿No puede ser simplemente como lo siguiente?
```suggestion
      <h3>{book.title}</h3>
```

Reply:
Si no me equivoco, es algo que corrige ESlint. No estoy seguro de cual es el error, pero se podría ignorar en esa línea